css/properties/background-clip.json

```diff
@@ -126,9 +126,11 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": "≤16"
+                "version_added": "4"
+              },
+              "firefox_android": {
+                "version_added": "14"
```

Comment:
I see this clamps to 14 because that's the earliest version for Firefox Android in the parent feature. But that seems fairly likely to be wrong. It comes from https://github.com/mdn/browser-compat-data/pull/76 which I assume was wiki migration, and there's no discussion of it.

I suggest instead mirroring both the parent feature and this feature, here and elsewhere that "14" appears in this PR.

Reply:
I thought it seemed odd, but I didn't look into it further -- thanks for confirming it was wiki migration data!  Opened #22541 and #22543 to address that!

Reply:
I've merged those two PRs, can you rebase this so it's easy to see the remaining changes?

Reply:
Not just rebase, changes some things to "mirror" too.